### PR TITLE
[3.4] member replace e2e test

### PR DESCRIPTION
**Pull request by ZhouJianMS**
Follow up: https://github.com/etcd-io/etcd/pull/17079#issuecomment-1856089798

### CODE REVIEW

**Comment by ZhouJianMS** [comment]
Does anyone know why 3.4 branch always show that "no tests to run" in local test? I have not encountered similar issue in 3.5 and main branch. 

```
Running tool: /usr/bin/go test -timeout 30m -tags e2e -run ^TestMemberReplace$ go.etcd.io/etcd/tests/v3/e2e -count=1

testing: warning: no tests to run
PASS
ok      go.etcd.io/etcd/tests/v3/e2e    0.014s [no tests to run]
```

**Comment by fuweid** [comment]
@ZhouJianMS I think there is no `go.etcd.io/etcd/tests/v3`. You should use `go.etcd.io/etcd/tests/e2e`.

File: go.mod [hunk 1]
```diff
@@ -33,12 +33,12 @@ require (
...
-	golang.org/x/net v0.17.0
-	golang.org/x/sync v0.3.0
-	golang.org/x/sys v0.13.0
+	golang.org/x/crypto v0.15.0
+	golang.org/x/net v0.18.0
+	golang.org/x/sync v0.5.0
+	golang.org/x/sys v0.14.0
 	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2
-	google.golang.org/grpc v1.58.3
+	google.golang.org/grpc v1.59.0
```
Comment (fuweid) [inline go.mod:41]: this pull request requires to update deps? If not, please move the changes.

Comment (ZhouJianMS) [reply go.mod]: The deps updates were introduced by running "make" command locally. I have removed them but found that `TestWatchDelayForEvent` always timeout. I think my change does not have any impact on it.
Failed jobs: https://github.com/etcd-io/etcd/actions/runs/7469684403/job/20327175916?pr=17173

**Review by serathius — APPROVED** [review]

**Review by ahrtr — APPROVED** [review]
lgtm

Thanks

**Comment by fuweid** [comment]
I think it's about timeout. We need to increase `TIMEOUT` value for `test-e2e-release`

**Comment by serathius** [comment]
> I think it's about timeout. We need to increase `TIMEOUT` value for `test-e2e-release` from `30m` to `60m`

I would be opposed from blindly increasing the test timeout on pull request testing. I think the time of test execution is important part of user experience. For me even the 30 minutes is not great. If this is not an issue with this PR, but a problem of consistently hitting the limit, we should consider analysing the e2e test scenario and reducing the time.

**Comment by fuweid** [comment]
> I would be opposed from blindly increasing the test timeout on pull request testing.

Agree. But checked out all the failure cases and last 4 rounds were timeout issue. 
Last run was `ok  	go.etcd.io/etcd/tests/e2e	1796.555s`. It's close to 30m.
It still impacts user experience. It's kind of flakey.
We can consider to increase the timeout value, do some investigation, fix it and turn value back.
We can't ignore the fact that we run e2e cases one by one. More cases might increase runtime.
And Github CI VM looks unstable, at least to me.

**Review by fuweid — APPROVED** [review]
LGTM

**Pull request merged**